domain/machine/state/state_test.go

```diff
@@ -769,3 +769,30 @@ func (s *stateSuite) TestSetKeepInstanceNotFound(c *gc.C) {
 	err := s.state.SetKeepInstance(context.Background(), "666", true)
 	c.Assert(err, jc.ErrorIs, machineerrors.MachineNotFound)
 }
+
+func (s *stateSuite) TestSetLXDProfiles(c *gc.C) {
```

Comment:
**todo:** Add also a test with some existing profiles, and check that if we insert `{"existing_profile", "new_profile"}` the `new_profile` is also inserted.

Reply:
Done, see a2920161dc7071233e891ffb74d360088da68463.